ycms/cms/views/zentral.py

```diff
@@ -0,0 +1,51 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+from ..models import Ward , Patient, User
+from django.template import loader
+from django.shortcuts import render
+from django.http import Http404
+from django.shortcuts import get_object_or_404, render
+from django.views.generic import TemplateView
+
+
+class ZentralView(TemplateView):
+    """
+    View to see all wards data and add a new one
+    """
+    template_name = "zentral.html"
+
+    def newward(self, request):
+        """
+        This function returns a list of all wards,
+        as well as a form for creating a new one(not implemented yet)
+
+        :param request: The current request
+        :type request: ~django.http.HttpRequest
+
+        """
+        #ward list which acts as a iterable object
+        total_wards = Ward.objects.all()
+        #total number of wards
+        total_wards_count = Ward.objects.all().count()
+        #total number of beds
+        total_beds_count = wards_count*28
+        #total occupied beds
+        #total_occupied_beds =  #looping through each ward and multiply with Ward.occupied_beds method
+        #total available beds
+        #total_available_beds =  #looping through each ward and multiply with Ward.available_beds method
+        #total number of doctors
+        total_doctors_count = User.objects.filter(job_type__contains='DOCTOR').count()
```

Comment:
```suggestion
        total_doctors_count = User.objects.filter(job_type=job_types.DOCTOR).count()
```
(same for the Nurses)

For this to work, you need to
```python
from ..constants import job_types
```

THe point of these constants is so we don't have random hardcoded strings in our code.